common/flatpak-run.c

```diff
@@ -380,6 +380,15 @@ flatpak_run_add_environment_args (FlatpakBwrap    *bwrap,
       flatpak_bwrap_add_args (bwrap,
                               "--dev", "/dev",
                               NULL);
+
+      if (context->devices & FLATPAK_CONTEXT_DEVICE_USB)
+        {
+          g_info ("Allowing USB device access. Note: this bypasses the portal.");
```

Comment:
There is no portal, it can't bypass a portal.

Reply:
sorry it's a left over from the cherry-picking. Will remove.

Reply:
The g_info itself was fine. The note was just not true.

Reply:
ok then.